src/workerd/api/urlpattern.h

```diff
@@ -4,8 +4,12 @@
 
 #pragma once
 
+#include "ada.h"
+
 #include <workerd/jsg/jsg.h>
-#include <workerd/jsg/url.h>
+
+#include <optional>
+#include <string_view>
```

Comment:
```suggestion
#include <kj/array.h>
#include <kj/one-of.h>
#include <kj/string.h>

#include <string>
#include <string_view>
#include <utility> // For std::move
#include <vector>
```

For include-what-you-use reasons.